examples/basic_mysql_connection/index.js

```diff
@@ -4,17 +4,28 @@ var mysql = require('mysql');
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
 
 var connection = mysql.createConnection({
-	host: config.host,
+	 host: config.host,
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
+					'SELECT * FROM todos WHERE Done=1',
```

Comment:
Yeah, that's an application-level decision. But application decisions change - for example, I often write code like `if (todo_is_done) {}`, in which case any 'truthy' value (like 1, or 2, or 124, or "yes") will be the same.

So if my application has added other 'truthy' values in as `Done`, your query will fail. I would probably write this using `WHERE NOT Done=0`. That said, this is a bit confusing to read, so it's also not a perfect solution.